brayns/common/scene/Scene.h

```diff
@@ -249,7 +249,8 @@ class Scene : public BaseObject
      * @return the model that has been added to the scene
      */
     ModelDescriptorPtr load(Blob&& blob, const size_t materialID,
-                            Loader::UpdateCallback cb);
```

Comment:
maybe move the params before the callback? also doxygen is missing.
I was wondering also if the materialID could move into the params, but following for what it is used for it may even be a loader param, as it is only used by some loaders. So then maybe for one of the next reviews.